packages/components/src/components/tab-title/tab-title.scss

```diff
@@ -42,30 +47,33 @@
 }
 
 .scale-s {
+  --calcite-internal-tab-title-close-padding: var(--calcite-spacing-none);
+  --calcite-internal-tab-title-close-margin-start: var(--calcite-spacing-xxs);
+  --calcite-internal-tab-title-close-margin-end: var(--calcite-spacing-sm);
+
   .content {
     @apply text-n2h py-1;
   }
-  .x-button {
-    @apply w-5;
-  }
 }
 
 .scale-m {
+  --calcite-internal-tab-title-close-padding: var(--calcite-spacing-xxs);
+  --calcite-internal-tab-title-close-margin-start: var(--calcite-spacing-xxs);
+  --calcite-internal-tab-title-close-margin-end: var(--calcite-spacing-sm);
+
   .content {
     @apply text-n1h py-2;
   }
-  .x-button {
-    @apply w-7;
-  }
 }
 
 .scale-l {
+  --calcite-internal-tab-title-close-padding: var(--calcite-spacing-xxs);
+  --calcite-internal-tab-title-close-margin-start: var(--calcite-spacing-xs);
+  --calcite-internal-tab-title-close-margin-end: var(--calcite-spacing-sm);
```

Comment:
This value is consistent across scale, do we need a custom property here?

Reply:
Will clean up!